### ISSUE TITLE
prevent-abbreviations rule incorrectly interpreting master as referring to main

### ISSUE THREAD

**Issue by darovic** [sindresorhus/eslint-plugin-unicorn#1742]
I have an enum with the label `Master`, completely unrelated to source control. The rule is firing off, when it should (ideally) be contextually aware that it has nothing to do with source control (the whole master vs. main discussion in git).

The rule firing is `prevent-abbreviations` with the message: 
```The variable `Master` should be named `Main`. A more descriptive name will do too. (unicorn/prevent-abbreviations)XO(unicorn/prevent-abbreviations)```

The issue in context (this should be valid):
```js
const enum TalentRank {
	None,
	Novice,
	Expert,
	Master,
}
```

I'm supressing the error in this case, but I couldn't find it mentioned anywhere so figured I'd report it so you could decide if it's worth bothering with or if it's niche enough to just tell people to suppress it in documentation if they run into it.

EDIT: It's worth noting that it's not objecting to the same variable name in a type definition.

**Comment by fisker** [comment]
This is a XO issue, can you open an issue there instead?
